src/plugins/run_conditions.rs

```diff
@@ -0,0 +1,12 @@
+use bevy::ecs::event::{Event, EventReader};
+
+/* pub fn resource_available<T: Resource>(resource: Option<Res<T>>) -> bool {
+    resource.is_some()
+} */
+
+pub fn event_called<T: Event>(mut event_reader: EventReader<T>) -> bool {
```

Comment:
@MatthewGriffin is this needed because we don't know how many planets we want. Assuming game settings are determined before the game is ran -> spawns space stations then needs to determine number of planets to spawn.

Reply:
If we need to spawn more planets per station than can easily change the spawn_resource_planet fn to spawn multiple planets per station. 

I did the event approach since it seemed simpler to just pass the transform of the station that the planet needs to spawn near than it was to figure out a way to wait a frame for the stations to spawn so that you can grab them with a query.